docs-ref-conceptual/azure-cli-reference-for-IoT.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Azure CLI references for Azure IoT
+description: Azure CLI reference landing page for Azure IoT
+author: dbradish-microsoft
+manager: barbkess
+ms.devlang: azurecli
+ms.topic: reference
+ms.date: 06/05/2020
+ms.author: dbradish
+ms.service: azure-cli
+ms.devlang: azurecli
+---
+
+# Azure CLI for Azure IoT
+
+The Azure Command Line Interface ([Azure CLI](/cli/azure/what-is-azure-cli)) is a set of commands used to create and manage Azure resources.  It is available across many Azure services including Azure IoT.  There are over 100 references for Azure IoT giving you the ability to work effectively with IoT services from a command line.
+
+## References for IoT
+
+There are two type of Azure CLI references for IoT: released and extension.  Released references have been published as a permanent part of the Azure CLI.  Extension references haven't shipped as part of the CLI but run as CLI commands. With extensions, you gain access to experimental and pre-release commands.
```

Comment:
For IoT, the CLI IoT extension should be treated and referred to like a flagship tooling product from Azure IoT. It does have experimental and pre-release commands, but significant quantities of the IoT CLI extension are not in this form and instead in long-term use form.

Put another way, the IoT extension is a natural half of the IoT CLI experience and IoT CLI users and IoT automation scenarios will not get far without it. In fact, all the data-plane functionality of IoT exists in the IoT extension.